main.go

```diff
@@ -72,6 +79,19 @@ func main() {
 			ipfs, err = spawn(ctx)
 		case "local":
 			ipfs, err = http(ctx)
+		case "temp":
+			opts := []CfgOpt{
```

Comment:
I'd just apply these to _all_ temporary nodes (get rid of the options, hard-code this in `tmpNode`).